src/rtcModule/webrtc.cpp

```diff
@@ -582,6 +587,58 @@ void RtcModule::hangupAll(TermCode code)
     }
 }
 
+RemoteLogger *RtcModule::getRemoteLogger()
+{
+    return mRemoteLogger.get();
+}
+
+string RtcModule::getDeviceInfo()
+{
+    // UserAgent Format
+    // MEGA<app>/<version> (platform) Megaclient/<version>
+    std::string userAgent = mKarereClient.api.sdk.getUserAgent();
+
+    std::string androidId = "MEGAAndroid";
+    std::string iosId = "MEGAiOS";
+    std::string testChatId = "MEGAChatTest";
+    std::string syncId = "MEGAsync";
```

Comment:
Maybe it's useful to add the `MEGAchatQt` and `MEGAclc` user-agents too, in case of debugging calls with the example apps.